src/base/noscript.scss

```diff
@@ -8,6 +8,12 @@
 	display: block !important;
 }
 
+@media screen {
+	%noscript-screen-display-block-important {
```

Comment:
@nschonni I think what you mentioned is the opposite of what I was doing (and both CodePen examples).

SASS' documentation recommends against declaring a placeholder outside a media query, then extending it from inside media queries (since that technically wouldn't make any sense).

In the original version of this PR, my placeholder selector was declared inside a media query (like my first CodePen example), but the extend directives were situated outside of media queries. Since all of the extend directives were generated into the placeholder selector that was already inside a media query, it worked fine.

My second CodePen example (which I updated this PR to resemble) is still essentially doing the same thing. But it might arguably be better since in my first CodePen example, if the placeholder selector was never extended, the generated CSS would contain an empty media query. With the second example, there's no chance of an empty media query getting generated in the CSS if nothing attempts to extend it.